docs/runtime/management.adoc

```diff
@@ -573,6 +573,120 @@ Installs an application from url on all nodes.
 }
 ----
 
+=== Uninstall an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Uninstalls an application with specific application key.
+
+.Url
+ POST app/uninstall
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Start an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Starts an application with specific application key.
+
+.Url
+ POST app/start
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Stop an application
+
+image:xp-760.svg[XP 7.6.0,opts=inline] Stops an application with specific application key.
+
+.Url
+ POST app/stop
+
+.Params
+[cols="1,1,3", options="header"]
+|===
+|Param
+|Type
+|Description
+
+|`key`
+|String
+|Application key, for instance, `com.enonic.app.contentstudio`.
+|===
+
+.Response
+[source,plain]
+----
+No response body
+----
+
+=== Subscribe to Server-Sent Events
+
+image:xp-760.svg[XP 7.6.0,opts=inline] The client initiates the SSE connection by using the media type `text/event-stream` in the `Accept` header. Then it gets updates automatically without requesting the server.
+
+.Url
+ GET app/events
+
+Currently that API supports the following events:
+
+- `list` - returns the details of applications which already installed
+- `installed` - returns the details of just installed an application
+- `state` - returns the details of just started or stopped an application
+- `uninstalled` - returns the `key` of just removed an application
```

Comment:
These three lines are somewhat hard to understand.  Are we talking about the last application that was installed, all applications, or just one application?  -  I am guessing that these methods return info on the last application that was installed.  If so, this would be more understandable:
- `installed` - returns the details of the last applications that was installed
- `state` - returns the details of the application that was just started or stopped
- `uninstalled` - returns the `key` of the application that was just removed.

However, if the methods take a parameter, the text should be more like " ... of the specified application".

Reply:
Done